setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+(( EUID != 0 )) && exec sudo -- "$0" "$@"
```

Comment:
A summary comment on this line would be helpful for others not as comfortable in scripting.